src/script/page/LeftSidebar/panels/Conversations/ConversationsList.styles.tsx

```diff
@@ -20,17 +20,26 @@
 import {CSSObject} from '@emotion/react';
 
 export const headingTitle: CSSObject = {
-  color: 'var(--main-color)',
+  color: 'var(--text-input-label)',
   fontSize: 'var(--font-size-small)',
   fontWeight: 'var(--font-weight-semibold)',
   paddingBottom: '4px',
   paddingLeft: '16px',
   marginTop: '16px',
   marginBottom: '0',
   borderBottom: '1px solid var(--border-color)',
+  textTransform: 'uppercase',
 };
 
 export const conversationsList: CSSObject = {
   margin: 0,
   paddingLeft: 0,
 };
+
+export const noResultsMessage: CSSObject = {
+  marginTop: '16px',
+  marginBottom: '40px',
```

Comment:
Sure, but doesn't the `margin-block' help with the `writing-mode'? Apart from that, isn't it just a different way of writing (as would be `margin: 16px 0 40px 0`)?

Reply:
I got it, but what I meant was that `margin-block` becomes useful when working with the `writing-mode` property, otherwise it is just a matter of preference, right? Or maybe it's a common practice in the project? 

<img width="739" alt="Screenshot 2024-10-22 at 13 52 29" src="https://github.com/user-attachments/assets/a82e3d81-26bb-4073-80c2-f8cdfe8a0f40">

Reply:
okay, so let's let it be for now :)